k8s/lgtm.yaml

```diff
@@ -0,0 +1,33 @@
+# this is intended for testing / demo purposes only, not for production use
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: lgtm
+spec:
+  selector:
+    matchLabels:
+      app: lgtm
+  template:
+    metadata:
+      labels:
+        app: lgtm
+    spec:
+      containers:
```

Comment:
no ports are exposed. this is what I used last time: https://github.com/cyrille-leclerc/my-shopping-cart/blob/open-telemetry/k8s/lgtm.yaml

Reply:
updated, thank you